test/e2e/tests/confirmations/alerts/insufficient-funds.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { strict as assert } from 'assert';
+import FixtureBuilder from '../../../fixture-builder';
+import {
+  PRIVATE_KEY,
+  convertETHToHexGwei,
+  withFixtures,
+  WINDOW_TITLES,
+} from '../../../helpers';
+import { SMART_CONTRACTS } from '../../../seeder/smart-contracts';
+import { scrollAndConfirmAndAssertConfirm } from '../helpers';
+import {
+  TestSuiteArguments,
+  openDAppWithContract,
+} from '../transactions/shared';
+
+describe.only('Alert for insufficient funds', function () {
+  it.only('Shows an alert when the user tries to send a transaction with insufficient funds', async function () {
+    const nftSmartContract = SMART_CONTRACTS.NFTS;
+    const ganacheOptions = {
+      accounts: [
+        {
+          secretKey: PRIVATE_KEY,
+          balance: convertETHToHexGwei(0.0053),
```

Comment:
Might be worth adding a small comment above this line explaining this particular number, or putting it in a const for the same reason.

Reply:
done